pyproject.toml

```diff
@@ -0,0 +1,74 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "paddleocr"
+# After each version release, the version number needs to be incremented
+version = "2.8.0"
+description = "Awesome OCR toolkits based on PaddlePaddle(8.6M ultra-lightweight pre-trained model, support training and deployment among server, mobile, embedded and IoT devices)"
+authors = [
+    {name = "???", email = "???@???.???"},
+]
+maintainers = [
+    {name = "???", email = "???@???.???"},
+]
```

Comment:
@jzhang533 @GreatV Please reivew and make some suggestions about `author` and `maintainers`.

Reply:
https://github.com/PaddlePaddle/PaddleScience/blob/9d8496280ddd3344e43cd4ccf38859274a13d1cf/setup.py#L25

https://github.com/PaddlePaddle/Paddle/blob/ed5bc2e8c219a5bbbb048b1985371d4d852bdaf5/setup.py#L1833-L1836

maybe "paddlepaddle"

Reply:
align with PaddlePaddle/Paddle is OK for now.